src/memory/sparse.rs

```diff
@@ -79,6 +78,14 @@ impl<R: Register> Memory<R> for SparseMemory<R> {
         fill_page_data(self, addr, size, source, offset_from_addr)
     }
 
+    fn fetch_flag(&mut self, page: usize) -> Result<u8, Error> {
+        if page < RISCV_MAX_MEMORY / RISCV_PAGESIZE {
```

Comment:
replace all `RISCV_MAX_MEMORY / RISCV_PAGESIZE` to `MAX_PAGES`
```suggestion
        if page < MAX_PAGES {
```